manifests/params.pp

```diff
@@ -34,6 +33,14 @@
               $update_locale_pkg = 'libc-bin'
             }
           }
+          case $::operatingsystemrelease {
```

Comment:
You can use [versioncmp](https://docs.puppet.com/puppet/latest/function.html#versioncmp) to check for the OS version. Versioncmp() outputs 0 if the version is equal and 1 for newer, so `versioncmp($::operatingsystemrelease, '16.04') >= 0 ` should give you Ubuntu Xenial and newer and futureproof this.

A usage example: https://github.com/puppetlabs/puppetlabs-apache/blob/master/manifests/version.pp#L24